### PR TITLE
[DOCS] Classref updates: Color, RigidBody*

### PR DESCRIPTION
**Pull request by cbscribe**

### CODE REVIEW

File: doc/classes/Color.xml [hunk 1]
```diff
@@ -4,8 +4,8 @@
 		Color in RGBA format with some support for ARGB format.
 	</brief_description>
 	<description>
-		A color is represented as red, green and blue (r,g,b) components. Additionally, "a" represents the alpha component, often used for transparency. Values are in floating point and usually range from 0 to 1. Some methods (such as set_modulate(color)) may accept values &gt; 1.
-		You can also create a color from standardised color names with Color.ColorN (e.g. Color.green) or [method @GDScript.ColorN].
+		A color is represented by red, green, and blue [code](r, g, b)[/code] components. Additionally, [code]a[/code] represents the alpha component, often used for transparency. Values are in floating point and usually range from 0 to 1.  Some properties (such as [member CanvasItem.modulate] ) may accept values &gt; 1.
```
Comment (LikeLakers2) [inline doc/classes/Color.xml]: You have a double space between the third and fourth sentences.

You also have a unneeded space just after the link to `CanvasItem.modulate`.

Comment (cbscribe) [reply doc/classes/Color.xml]: Good catch, thanks.

**Comment by akien-mga** [comment]
Could you squash the commits together?

**Comment by cbscribe** [comment]
That'll teach me to try a new git tool. Branch is a tangled mess now. I'll fix with a new PR.

**Comment by akien-mga** [comment]
You could have force pushed the new branch on the remote one and kept the same PR :)

File: modules/bullet/rigid_body_bullet.h [hunk 2]
```diff
@@ -227,7 +227,7 @@ class RigidBodyBullet : public RigidCollisionObjectBullet {
 
 	void init_kinematic_utilities();
 	void destroy_kinematic_utilities();
-	_FORCE_INLINE_ KinematicUtilities *get_kinematic_utilities() const { return kinematic_utilities; }
+	_FORCE_INLINE_ class KinematicUtilities *get_kinematic_utilities() const { return kinematic_utilities; }
```
Comment (AndreaCatania) [inline modules/bullet/rigid_body_bullet.h]: Why do you added a class here?
KinematicUtilities is a struct and also is defined in this file

File: modules/bullet/space_bullet.h [hunk 3]
```diff
@@ -57,7 +57,7 @@ class btDiscreteDynamicsWorld;
 class btEmptyShape;
 class btGhostPairCallback;
 class btSoftRigidDynamicsWorld;
-struct btSoftBodyWorldInfo;
+class btSoftBodyWorldInfo;
```
Comment (AndreaCatania) [inline modules/bullet/space_bullet.h]: Can you please explain why class should be used over struct?

File: modules/gdscript/gdscript_parser.cpp [hunk 4]
```diff
@@ -7835,7 +7835,7 @@ void GDScriptParser::_check_block_types(BlockNode *p_block) {
 							// Figure out function name for warning
 							String func_name = _find_function_name(op);
 							if (func_name.empty()) {
-								func_name = "<undetected name>";
+								func_name == "<undetected name>";
```
Comment (AndreaCatania) [inline modules/gdscript/gdscript_parser.cpp]: This is not my scope but at first look seems to me that this line is a mistake

**Comment by akien-mga** [comment]
@AndreaCatania This was the result of a wrong git rebase, and all things you commented on are actually bugs I fixed (but the other way around compared to this diff :)).

**Comment by akien-mga** [comment]
@cbscribe Now you know why merge commits should be rebased away but never reverted :P
screenshot_20181003_095750

**Comment by AndreaCatania** [comment]
@akien-mga you are always ahead of the game. I like it :)

**Comment by cbscribe** [comment]
Thanks - I got lost in a loop of trying to fix it and making it worse. Sorry for the confusion.

**Pull request closed without merging**